priam/src/main/java/com/netflix/priam/notification/BackupNotificationMgr.java

```diff
@@ -77,7 +81,8 @@ public void notify(BackupVerificationResult backupVerificationResult) {
             jsonObject.put("snapshotInstant", backupVerificationResult.snapshotInstant);
```

Comment:
As long as you're here, let's pay down some debt. Please convert the SNAPSHOT_VERIFIED string literal in line 80 to AbstractBackupPath.BackupFileType.SNAPSHOT_VERIFIED.name().